src/main/java/core/basesyntax/handler/impl/ReturnOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.handler.impl;
+
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+
+public class ReturnOperationHandler implements OperationHandler {
```

Comment:
The class name `ReturnOperationHandler` is not following Java class naming conventions. Consider renaming it to match the pattern of `VerbNoun` format, which could be something like `HandleReturnOperation`.